torchgeo/models/earthloc.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# https://github.com/gmberton/EarthLoc
```

Comment:
Available under an MIT license, which is perfect for us. Note that if any of the below code was directly copied from the other repo, we need to add "Copyright (c) 2024 Gabriele Berton" to that code.

Reply:
Both are okay with me, as long as the copied parts are under the right copyright.